Userland/test-crypto.cpp

```diff
@@ -99,7 +99,7 @@ int crc32_tests();
 
 // stop listing tests
 
-void print_buffer(const ByteBuffer& buffer, int split)
+void print_buffer(const Span<const u8>& buffer, int split)
```

Comment:
Seems like this should be `ReadonlyBytes`

Reply:
_/me patches stuff and doesn't add it before amending_